internal/service/searchindex/model_search_index.go

```diff
@@ -0,0 +1,151 @@
+package searchindex
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"log"
+	"strconv"
+
+	"github.com/go-test/deep"
```

Comment:
Any reason for using this instead of `reflect.DeepEqual` ?

Reply:
i didn't change the current code, just moved to another file, will investigate if DeepEqual can be used

Reply:
@EspenAlbert ticket created CLOUDP-261460 as it's being used in more places